doc/fitting.rst

```diff
@@ -15,59 +15,17 @@ when using complicated constraints or comparing results from related fits.
 
 
 The :func:`minimize` function
-===============================
+=============================
+
+.. currentmodule:: lmfit.minimizer
```

Comment:
please leave as:

    ..currentmodule:: minimizer

This makes the intersphinx references uniform across parameters.rst and model.rst. For example, the intersphinx reference for `Parameters` is currently `lmfit::Parameters` and `minimize` is `lmfit::minimize`.
If this change is kept then the intersphinx reference becomes `lmfit::lmfit.minimize`.